client/CBitmapHandler.cpp

```diff
@@ -42,33 +42,36 @@ SDL_Surface * BitmapHandler::loadH3PCX(ui8 * pcx, size_t size)
 	SDL_Surface * ret;
 
 	Epcxformat format;
-	int it=0;
-
-	ui32 fSize = read_le_u32(pcx + it); it+=4;

```

Comment:
What is current opinion on "never keep multiple statements in one line?" I wonder if these "multiple statements" dont actually improve code readability when used smartly.